components/raftstore-v2/src/operation/query/mod.rs

```diff
@@ -71,6 +74,151 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: raftstore::store::Transport>
         }
     }
 
+    #[inline]
+    pub fn on_refresh_region_buckets(
```

Comment:
Could it be moved to `Peer` instead?

Reply:
This is not a query, the function and other related functions should be put to `operation/bucket.rs`.

Reply:
yes, It will moved to `operation/bucket.rs`.